mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -279,11 +285,16 @@ def start_conn_process():
                             self.logger.prn_wrn("stopped to consume events due to %s event"% key)
                             callbacks_consume = False
                             result = self.RESULT_IO_SERIAL
-                            break
+                            event_queue.put(('__exit_event_queue', 0, time()))
```

Comment:
How does it improve on break. If handling of ```__exit_event_queue``` just sets ```callbacks__exit_event_queue = True``` then that can be done here!